db/migrate/20240130013701_create_post_action_reaction_user_historical_records.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class CreatePostActionReactionUserHistoricalRecords < ActiveRecord::Migration[7.0]
+  def up
```

Comment:
oh ... the first sync is here... I see...

Reply:
Yeah I thought this would be best. It did just occur to me though that maybe this would be better as a post migration.